operatorcourier/api.py

```diff
@@ -5,16 +5,16 @@
 """
 
 import os
-import logging
+import logging.config
 from tempfile import TemporaryDirectory
 import yaml
 from operatorcourier.build import BuildCmd
 from operatorcourier.validate import ValidateCmd
 from operatorcourier.push import PushCmd
 from operatorcourier.format import format_bundle
 
-logging.basicConfig()
-logger = logging.getLogger(__name__)
+logging.config.fileConfig("./config/logging.ini")
+logger = logging.getLogger()
```

Comment:
@MartinBasti @kevinrizza 

Do you know of an easy way to set the default logging configuration without adding a logger within the `config/logging.ini` for each module?

Reply:
Oh wait, you are not supposed to configure logging in python modules which are supposed to by used by others. Logging should be configured only in apps (you can configure logging operator-courier CLI main function, not in API).

`operator-courier/operatorcourier/cli.py::main`  is the right place where logging should be configured

You don't have to add logging config to each module. It is enough to just configure root logger in main (`logging.basicConfig(...)`) or you can use logger hierarchy. configuration of logger for `operatorcourier` will have effect for all subloggers `operatorcourier.*` (example `logging.getLogger('operatorcourier.api')`).  With using `__name__` as loggername, logger names should already have format of `operatorcourier.module`

Reply:
Oh I see you are using root logger already, so just move it to `main`